spec/factories/orders.rb

```diff
@@ -0,0 +1,8 @@
+require 'faker'
+FactoryBot.define do
+  factory :order do
+    product_name { Faker::Commerce.product_name }
+    product_count { Faker::Number.between(from: 1, to: 100) }  # Ensures product_count is a realistic number
```

Comment:
Faker's a lot of fun! But just for fun, look up how to make a random number with just Ruby :)